gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/GradebookPage.java

```diff
@@ -280,6 +282,39 @@ public String getCssClass() {
 		};
 		cols.add(studentNameColumn);
 
+		// student extra info optional column
```

Comment:
Not sure if this is still true in master, but adding a column here in Sakai 11 messes up some of the javascript and css that expect only the first two columns to need to be "frozen". Also, with categories on I had to adjust the "empty" cell's colspan to accomodate for the extra column.